src/bitmessageqt/messageview.py

```diff
@@ -68,35 +80,33 @@ def confirmURL(self, link):
             )
             window.ui.textEditMessage.setFocus()
             return
-        reply = QtGui.QMessageBox.warning(self,
-            QtGui.QApplication.translate("MessageView", "Follow external link"),
-            QtGui.QApplication.translate("MessageView", "The link \"%1\" will open in a browser. It may be a security risk, it could de-anonymise you or download malicious data. Are you sure?").arg(unicode(link.toString())),
-            QtGui.QMessageBox.Yes, QtGui.QMessageBox.No)
+        reply = QtGui.QMessageBox.warning(
+            self,
+            QtGui.QApplication.translate(
+                "MessageView",
+                "Follow external link"),
+            QtGui.QApplication.translate(
+                "MessageView",
+                "The link \"%1\" will open in a browser. It may be a security risk, it could de-anonymise you"
+                " or download malicious data. Are you sure?").arg(unicode(link.toString())),
+            QtGui.QMessageBox.Yes,
+            QtGui.QMessageBox.No)
         if reply == QtGui.QMessageBox.Yes:
             QtGui.QDesktopServices.openUrl(link)
 
-    def loadResource (self, restype, name):
-        if restype == QtGui.QTextDocument.ImageResource and name.scheme() == "bmmsg":
-            pass
-#            QImage correctImage;
-#            lookup the correct QImage from a cache
-#            return QVariant::fromValue(correctImage);
-#        elif restype == QtGui.QTextDocument.HtmlResource:
-#        elif restype == QtGui.QTextDocument.ImageResource:
-#        elif restype == QtGui.QTextDocument.StyleSheetResource:
-#        elif restype == QtGui.QTextDocument.UserResource:
-        else:
-            pass
-#            by default, this will interpret it as a local file
-#            QtGui.QTextBrowser.loadResource(restype, name)
+    def loadResource(self, restype, name):
+        """"""
+        pass
 
     def lazyRender(self):
+        """"""
```

Comment:
```suggestion
        """Partially render a message. This is to avoid UI freezing when loading huge messages. It continues loading as you scroll down."""
```